src/components/calcite-accordion/readme.md

```diff
@@ -0,0 +1,28 @@
+# calcite-dropdown
```

Comment:
calcite-accordion here, also would be good to write a quick blurb about what it's for, why to use it, single copy paste example